arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -62,6 +60,7 @@ public class RomLex
       new ModuleOperationStackedSet<>();
 
   @Getter private List<RomOperation> sortedOperations;
+  Map<ContractMetadata, List<RomOperation>> lookup = new HashMap<>();
```

Comment:
We have a bijection between contract metadata and one RomOperation, not a list of RomOperations